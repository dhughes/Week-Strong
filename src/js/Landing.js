import React from 'react';

const LoggedOutLanding = () => <div>Logged in</div>;

export default LoggedOutLanding;
