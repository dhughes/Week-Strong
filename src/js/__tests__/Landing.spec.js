import React from 'react';
import { shallow } from 'enzyme';
import Landing from '../Landing';

test('Landing renders', () => {
  const component = shallow(<Landing />);
  expect(component).toMatchSnapshot();
});
