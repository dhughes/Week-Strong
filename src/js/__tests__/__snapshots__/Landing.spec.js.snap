// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing renders 1`] = `
<div>
  Logged in
</div>
`;
